### PR TITLE
Nisse/v0.5.3/metadata on sensorframe

### PR DESCRIPTION
**Pull request by pd-nisse**

### CODE REVIEW

File: paralleldomain/decoding/dgp/v1/decoder.py [hunk 1]
```diff
@@ -197,6 +196,8 @@ def _create_frame_decoder(self, scene_name: SceneName, frame_id: FrameId, datase
 
     @lru_cache(maxsize=1)
     def _decode_scene_dto(self, scene_name: str) -> scene_pb2.Scene:
+        import paralleldomain.common.dgp.v1.metadata_pd_pb2  # noqa: F401
```
Comment (phillip-thomas-pd) [inline paralleldomain/decoding/dgp/v1/decoder.py:199]: why import here?

File: paralleldomain/decoding/dgp/v1/decoder.py [hunk 2]
```diff
@@ -138,10 +137,10 @@ def _decode_frame_id_to_date_time_map(self, scene_name: SceneName) -> Dict[Frame
 
     def _decode_set_metadata(self, scene_name: SceneName) -> Dict[str, Any]:
         scene_dto = self._decode_scene_dto(scene_name=scene_name)
-        metadata = {k: v for k, v in scene_dto.metadata.items()}
-        if "PD" in metadata:
-            metadata["PD"] = dict(MessageToDict(scene_dto.metadata["PD"], ParallelDomainSceneMetadata()))
-        return metadata
+        # metadata = {k: v for k, v in scene_dto.metadata.items()}
```
Comment (phillip-thomas-pd) [inline paralleldomain/decoding/dgp/v1/decoder.py:140]: do we still need this code?

File: paralleldomain/decoding/dgp/v1/decoder.py (hunk 1 above)
Comment (pd-nisse) [reply paralleldomain/decoding/dgp/v1/decoder.py:199]: protobuf "any" requires that the used schemas are loaded when parsing the scene.json into a pb2 object. I could have left it at the top of the file, but decided to put it in place where it is needed so it doesnt get actually "optimized away" by some unused imports feature in Pycharm etc.

File: paralleldomain/decoding/dgp/v1/decoder.py (hunk 2 above)
Comment (pd-nisse) [reply paralleldomain/decoding/dgp/v1/decoder.py]: no :grin: (removed it)

**Review by phillip-thomas-pd — APPROVED** [review]

**Pull request merged**